### PR TITLE
Feature/java11 neo4jv4

### PR DESCRIPTION
**Pull request by DirkMahler**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -80,8 +80,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <doxia-markdown.version>1.9</doxia-markdown.version>
-        <com.buschmais.xo.version>1.4.2</com.buschmais.xo.version>
-        <org.neo4j_3x.version>3.5.35</org.neo4j_3x.version>
+        <com.buschmais.xo.version>2.0.0-SNAPSHOT</com.buschmais.xo.version>
```
Comment (StephanPirnbaum) [inline pom.xml]: Can we build a release of XO first?

File: status.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Java 11
```
Comment (StephanPirnbaum) [inline status.txt]: Please remove this file

**Review by StephanPirnbaum — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (DirkMahler) [reply pom.xml]: Deferred 2.0.0 release of XO until no further changes are required for Neo4j v4 migration

File: status.txt (hunk 2 above)
Comment (DirkMahler) [reply status.txt]: Removed

File: pom.xml [hunk 3]
```diff
@@ -646,7 +646,7 @@
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
-             <dependency>
+                 <dependency>
```
Comment (obfischer) [inline pom.xml]: Wrong indentation.

File: pom.xml (hunk 1 above)
Comment (obfischer) [reply pom.xml]: I agree with @StephanPirnbaum, but for me it isn't a blocker.

**Review by obfischer — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (DirkMahler) [reply pom.xml]: Fixed

File: pom.xml (hunk 1 above)
Comment (DirkMahler) [reply pom.xml]: Upgraded to XO 2.0.0

**Review by StephanPirnbaum — APPROVED** [review]

**Pull request merged**